backend/account/views.py

```diff
@@ -73,6 +73,42 @@ def update_account(request: HttpRequest):
     return JsonResponse(status=200, data={})
 
 
+@require_POST
+def update_privacies(request: HttpRequest):
+    """
+    Updates privacy settings given a user id
+    """
+
+    body = read_request(request)
+    logger.debug(f"update_privacies: {id}")
```

Comment:
Looks like this `{id}` isn't defined

Reply:
I actually copied this from the update_account method. Will attempting to get the field be the wrong way to do it? Could raise an exception...

Reply:
`update_user` has an `id` field, vs it looks like you've used a `user_id` field for `update_privacies` which is likely what caused the error i noticed in logs